### ISSUE TITLE
Not able to get 82.1 acc@1 for ConvNeXt-T (single machine training for ImageNet-1k) 

### ISSUE THREAD

**Issue by shantanuj** [facebookresearch/ConvNeXt#87]
Hi, thanks for the great work and for sharing the codebase!

I was trying to reproduce results and trained a ConvNeXt-T on a single machine across 4 GPUs, individual batch size 64 and update frequency 16 (effectively 4096 batch size). I kept every other setting the same as mentioned in TRAINING.md. However, I am only able to get 81.47% top1 (without amp training) and 81.35% (with amp). I also checked and found that the provided pre-trained models have an 82.12% top1, indicating that my validation set is the same. 

Just wondering whether this is expected / perhaps there's some setting I haven't enabled?

Thanks!

**Comment by shantanuj** [comment]
Ah got it, thanks for the helpful comment. I'll try to see if there's a workaround for smaller batch sizes.
